src/client/sandbox/node/window.js

```diff
@@ -420,6 +430,56 @@ export default class WindowSandbox extends SandboxBase {
             };
         }
 
+        if (window.WebSocket) {
+            window.WebSocket = function (url, protocols) {
+                if (arguments.length === 0)
+                    return new nativeMethods.WebSocket();
+
+                const proxyUrl = getProxyUrl(url, { resourceType: stringifyResourceType({ isWebSocket: true }) });
+                let webSocket  = null;
+
+                if (arguments.length === 1)
+                    webSocket = new nativeMethods.WebSocket(proxyUrl);
+                else if (arguments.length === 2)
+                    webSocket = new nativeMethods.WebSocket(proxyUrl, protocols);
+                else
+                    webSocket = new nativeMethods.WebSocket(proxyUrl, protocols, arguments[2]);
+
+                // NOTE: We need to use deprecated methods instead of the defineProperty method in Android 5.1
+                // because the defineProperty does not redefine descriptor
```

Comment:
Add a one more sentence: `Try to drop this after moving to a higher Android version`